### PR TITLE
Move email sending from versionbot to CI

### PR DESCRIPTION
**Pull request by anujdeshpande**
See related PR - https://github.com/product-os/versionbot/pull/403

### CODE REVIEW

**Comment by anujdeshpande** [comment]
https://github.com/product-os/versionbot/pull/403

File: github/store-github-release.sh [hunk 1]
```diff
@@ -24,8 +24,24 @@ ASSETS=$(find "outputs/" -type f)
...
   -b ${headBranch} \
   -r ${repo} \
   -o ${org} \
-  -v ${version}
+  -v ${version} > output.txt
+
+cat output.txt
+RELEASE_URL=$( grep -nri -A 10 'Published draft release:' output.txt | grep html_url -A 1 | grep -e "[\'\"]" )
+
+echo $RELEASE_URL
```
Comment (stathismor) [inline github/store-github-release.sh:37]: How about a more descriptive log, like "Sending email with subject blah to blah" or whatever you think is valuable.

File: hardware/email.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Example - {"ownerName":"balena-io-hardware", "repoName":"hardware-process-test", "branch":"dum-dum-pr", "installationId":15652884}
```
Comment (stathismor) [inline hardware/email.sh:1]: Remind me, is this file used? Where? Subject seems a test one...

File: github/store-github-release.sh [hunk 3]
```diff
@@ -24,8 +24,24 @@ ASSETS=$(find "outputs/" -type f)
...
+# Example RELEASE_URL - 'https://github.com/balena-io-hardware/hardware-process-test/releases/tag/untagged-0b077a1f62efcb13db09'
 resinci-deploy store github-release "${ASSETS}" \
   -b ${headBranch} \
   -r ${repo} \
   -o ${org} \
-  -v ${version}
+  -v ${version} > output.txt
+
+cat output.txt
+RELEASE_URL=$( grep -nri -A 10 'Published draft release:' output.txt | grep html_url -A 1 | grep -e "[\'\"]" )
```
[the remark this answers is not in the document]
Comment (anujdeshpande) [reply github/store-github-release.sh]: Done

File: hardware/email.sh (hunk 2 above)
Comment (anujdeshpande) [reply hardware/email.sh]: yeah that file isn't used anymore as we've put that in the github release file. forgot i had that here. removed it.

**Review by stathismor — APPROVED** [review]

**Pull request merged**